modules/global-classes/module.php

```diff
@@ -37,6 +38,10 @@ public function __construct() {
 
 			( new Global_Classes_REST_API() )->register_hooks();
 			( new Global_Classes_CSS() )->register_hooks();
+
+			if ( is_admin() ) {
```

Comment:
this is just hooks, do we need to check `is_admin`?

Reply:
it was like this in the kit defaults, but removed